TypeCobol/Compiler/Directives/IBMCompilerOptions.cs

```diff
@@ -546,6 +551,9 @@ public class IBMCompilerOptions
         // Conversion of option words in source text to option name enumeration
         private readonly IDictionary<string, IBMCompilerOptionStatus> optionWordToOptionName = new Dictionary<string, IBMCompilerOptionStatus>(63);
               
+        // Deprecated options which are not supported anymore with the corresponding warning message to be displayed
+        private readonly IDictionary<string, string> deprecatedOptions = new Dictionary<string, string>(1);
```

Comment:
This should be a `static` dictionary. Also you should set the `StringComparer.OrdinalIgnoreCase` in the constructor to avoid having to call `ToUpper`.

Reply:
Done